pkgs/development/compilers/gcc/default.nix

```diff
@@ -408,14 +409,28 @@ lib.pipe ((callFile ./common/builder.nix {}) ({
       maintainers
     ;
   } // lib.optionalAttrs (!atLeast11) {
-    badPlatforms = if !is49 then [ "aarch64-darwin" ] else lib.platforms.darwin;
+    badPlatforms = if !(is48 || is49) then [ "aarch64-darwin" ] else lib.platforms.darwin;
   };
-} // optionalAttrs is7 {
-  env.NIX_CFLAGS_COMPILE = lib.optionalString (stdenv.cc.isClang && langFortran) "-Wno-unused-command-line-argument";
```

Comment:
Why was this deleted?

Reply:
It was consolidated with the other change to `NIX_CFLAGS_COMPILE`.

Reply:
Yeah; unfortunately `drv.NIX_CFLAGS_COMPILE=""` and `!(drv?NIX_CFLAGS_COMPILE)` hash differently.